Network/HTTP/Req.hs

```diff
@@ -1297,13 +1301,24 @@ instance HttpBody ReqBodyUrlEnc where
 
 -- | An opaque monoidal value that allows to collect URL-encoded parameters
 -- to be wrapped in 'ReqBodyUrlEnc'.
+--
+-- You can inspect its content with 'toList'.
 newtype FormUrlEncodedParam = FormUrlEncodedParam [(Text, Maybe Text)]
   deriving (Semigroup, Monoid)
 
 instance QueryParam FormUrlEncodedParam where
   queryParam name mvalue =
     FormUrlEncodedParam [(name, toQueryParam <$> mvalue)]
 
+-- | Use 'formToQuery'.
+instance FromForm FormUrlEncodedParam where
```

Comment:
I think it makes sense to also define `ToForm`?

Reply:
Internally, `FormUrlEncodedParam = [(Text, Maybe Text)]` while `Form = HashMap Text [Text]`. So, `FormUrlEncodedParam` is more expressive than `Form`. `toForm` would result in loss of information in some cases, e.g., there is no counterpart `Form` for `FormUrlEncodedParam = [("a", Nothing), ("a", Nothing), ("a", Nothing)]`. That was why I didn't add `ToForm` instance.

Reply:
Oops, sorry, I was wrong. `FormUrlEncodedParam  [("a", Nothing), ("a", Nothing), ("a", Nothing)]` can be expressed by `Form $ fromList [("a", ["", "", ""])]`. However, `FormUrlEncodedParam [("a", Just "")]` cannot be expressed.